### ISSUE TITLE
Autosmooth menu not working properly in blender 4.1

### ISSUE THREAD

**Issue by AMYO77** [1C0D/Autosmooth-Menu-BlenderAddon#1]
Sorry to bother you, they changed the name of autosmooth to "shade smooth by angle", could you update it please ?

**Comment by 1C0D** [comment]
you should do a pull request

**Comment by AMYO77** [comment]
i'm sorry to bother you again, i cant find a way to make a pull request, 
![Capture d’écran 2024-02-21 145625](https://github.com/1C0D/Autosmooth-Menu-BlenderAddon/assets/22004001/d0ca834a-dd18-4431-ac98-2a94ad802b5c)

**Comment by 1C0D** [comment]
ah you have to fork the repo. clone the new repo on your computer use git after changes and do git add . then git commit -m "title of your changes" then git push.
you could ask to GPT to assist you, if you want to learn. But if it's too difficult you could just paste the code to change there. I watched but I even don't remember how to do things under Blender. 2 years without touching it and all is vanished. Else, maybe I take a look later. But I'm really not motivated to get back in. I don't like the spirit of the team, considering coding. jerks... Now I code in Obsidian some plugins and that's much cleaner, I don't feel some people who want to make money without really telling it. it's just a share of knowledge and the team who did earn money with it. it's clear. The blender boss received millions 2 years ago. for what ? I wonder. Maybe not to make it evolve to fast or not at all. Under the surface it's shitty. Well that's my perception so I don't enjoy it at all since...

**Comment by AMYO77** [comment]
I agree with you and i can totally understand your point. 
To me what's making blender better is devs like you who really had something to the community.
I tried to do it myself thinking, it could be just looking for the right command and copy paste, but i'm noob in this so i gave up.
i rely a lot on this tool so i'm sad to see it abandonned but i'll respect your choice. Thanks again for your work.

**Comment by 1C0D** [comment]
ah you seem really kind so maybe I do an effort to get back in, and see how to code a plugin again. It should come back quickly. And python too... I even don't remember where I was chatting with other devs...
I watched 2 episodes of Blender Guru yesterday after your issue. And for instance, the first thing I did on blender was changing G into click to directly move vertices. and drag right click to select box. but 20 years later (Guru said he was using it since 20 years...) they still didn't find a better way to do things. I'm speechless... And often he is telling he doesn't like how something works, because it's not ergonomic, and I know by experience that the devs don't care... but he does the job...
And BTW the hotkey map is a total mess. I'm not a "good" programmer, but wow if you try obsidian, you will see the difference, it's super clean and everybody can handle it. In a 3D software, it should be an absolute priority.

**Comment by 1C0D** [comment]
I see there is an advanced autosmooth now. Maybe this is what you are telling this is not working the same way?
![image](https://github.com/1C0D/Autosmooth-Menu-BlenderAddon/assets/63018905/bb0b7e31-f968-40c4-8d68-008138f25309)

**Comment by AMYO77** [comment]
![Capture d’écran 2024-02-21 192637](https://github.com/1C0D/Autosmooth-Menu-BlenderAddon/assets/22004001/e9660f00-2fe3-4f41-b97d-beb69af18fc2)

I'm talking about blender 4.1 since it will be out in few weeks, i'm looking at what's working and what's not.
The autosmooth menu itself is working but for me when i want to clic on the box to activate it, it stay silent and doesnt do anything. Everything is fine in blender 4.0.

But i know they changed the autosmooth command, now it's a command that you can access in the right clic menu and it's a geo node modifier.

**Comment by 1C0D** [comment]
oh guess what I installed LTS version so 4.0.0 that's why. chease... will see

**Comment by 1C0D** [comment]
yes indeed we answered at the same moment

**Comment by AMYO77** [comment]
yeah you're right, i'm looking at how they changed it, i'm not sure updating the command or making it add the geo node setup is a good idea, the basic idea is to use the geo node modifier for export / import in game engine. 
I'm sorry, just removing the checkbox should be enough, i should've not bother you with so little.
It's still usable in his current state.

**Comment by 1C0D** [comment]
hmm you don't to apologize at all. I will see this later again
